lib/tests/streamlit/config_test.py

```diff
@@ -28,10 +28,11 @@
 from streamlit import env_util
 from streamlit.ConfigOption import ConfigOption
 
+os.environ["STREAMLIT_COOKIE_SECRET"] = "chocolatechip"
+
 SECTION_DESCRIPTIONS = copy.deepcopy(config._section_descriptions)
 CONFIG_OPTIONS = copy.deepcopy(config._config_options)
 
-
```

Comment:
Sanity-checking whether deleting this whitespace was intentional.  PEP-8 recommends 

> Surround top-level function and class definitions with two blank lines.

 https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
not intentional. i just assume we had auto-pep8 that will fix any spacing issues